login-workflow/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Fixed
 
+-   Fixed Publishing Happens before Tests Finish in the circleci config file ([#202](https://github.com/etn-ccis/blui-angular-workflows/issues/202)).
```

Comment:
Please remove this from the changelog. This is not a change happened to the workflow itself; our users does not need to know about this change.